portfolio/src/main/java/com/google/sps/containers/Comment.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps.containers;
+
+public final class Comment{
```

Comment:
Can you add Javadoc to this class (and the new classes below) describing what they're for?

Reply:
Done.